servicetalk-dns-discovery-netty/src/main/java/io/servicetalk/dns/discovery/netty/DefaultDnsClient.java

```diff
@@ -813,9 +813,11 @@ private void handleResolveDone0(final Future<DnsAnswer<T>> addressFuture,
                     final List<T> addresses = dnsAnswer.answer();
                     ttlNanos = dnsAnswer.ttlNanos();
                     if (ttlNanos > maxTTLNanos) {
-                        LOGGER.info("{} result for {} has a high TTL={}s which is larger than configured maxTTL={}s.",
-                                DefaultDnsClient.this, AbstractDnsPublisher.this,
-                                NANOSECONDS.toSeconds(ttlNanos), NANOSECONDS.toSeconds(maxTTLNanos));
+                        if (LOGGER.isDebugEnabled()) {
+                            LOGGER.debug("{} result for {} has a high TTL={}s which is larger than configured maxTTL={}s.",
```

Comment:
Ya know, putting `s` on the numeric value makes it harder to parse. I might recommend the unit be part of the field name.

Reply:
I'll just remove the units. TTL is always in seconds for DNS.